numba_dppy/extended_numba_itanium_mangler.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from numba.core import itanium_mangler, types
+from numba_dppy import target
+
+EXQ2ABI = {
```

Comment:
Add a comment stating what the dictionary stores.

Reply:
I think we don't need to hardcode this values. The Itanium ABI specifies `address_space` to be mangled as `AS` and the `address_space` is assigned to the pointer so we can extract it from there.